examples/siro_sandbox/sandbox_service.py

```diff
@@ -88,3 +94,15 @@ def end_episode(self):
     @property
     def set_cursor_style(self):
         return self._set_cursor_style
+
```

Comment:
This is so verbose... three new members just to make this episode info available... I'm worried SandboxService is going to become very bloated. I guess this is fine for now. In the future, maybe we need to find a more elegant and scalable way to expose things from SandboxDriver to AppStates.

Reply:
If you're feeling ambitious: I suppose an EpisodeHelper class, that SandboxDriver owns, and then SandboxService exposes this object. So we only have one new member in SandboxService instead of three. Maybe that is cleaner.

Reply:
done